defold-rive/pluginsrc/plugin.cpp

```diff
@@ -33,21 +33,19 @@ __declspec(dllexport) int dummyFunc()
 #include <dmsdk/dlib/array.h>
 #include <dmsdk/dlib/log.h>
 #include <dmsdk/dlib/shared_library.h>
+#include <dmsdk/dlib/static_assert.h>
 #include <stdio.h>
 #include <stdint.h>
 
 #include <common/bones.h>
+#include <common/vertices.h>
 
-struct RiveBuffer
-{
-    void*        m_Data;
-    unsigned int m_Size;
-};
+// We map these to ints on the java side (See Rive.java)
+DM_STATIC_ASSERT(sizeof(dmGraphics::CompareFunc) == sizeof(int), Invalid_struct_size);
+DM_STATIC_ASSERT(sizeof(dmGraphics::StencilOp) == sizeof(int), Invalid_struct_size);
+// Making sure the size is the same for both C++ and Java
+DM_STATIC_ASSERT(sizeof(dmRive::RenderObject) == 288, Invalid_struct_size);
```

Comment:
I guess we could add asserts for the offsets of the fields as well, to make it even more robust.